### PR TITLE
Pagerduty adjustments

### PR DESCRIPTION
**Pull request by fbalak**

### CODE REVIEW

File: tests/manage/monitoring/pagerduty/test_deployment_status.py [hunk 1]
```diff
@@ -33,17 +34,20 @@ def test_ceph_manager_stopped_pd(measure_stop_ceph_mgr):
...
-    )
-    api.check_incident_cleared(
-        summary=target_label, measure_end_time=measure_stop_ceph_mgr.get("stop")
-    )
+        # TODO(fbalak): check the whole string in summary and incident alerts
+        assert pagerduty.check_incident_list(
+            summary=target_label,
+            incidents=incidents,
+            urgency="high",
+        )
```
Comment (mbukatov) [inline tests/manage/monitoring/pagerduty/test_deployment_status.py:47]: When this fails, would you learn which alert is causing the problem? Also, would it be useful to know when both alerts are missing?

Comment (fbalak) [reply tests/manage/monitoring/pagerduty/test_deployment_status.py:47]: This can be read from logs. In logs are listed all alerts that were triggered.

Comment (mbukatov) [reply tests/manage/monitoring/pagerduty/test_deployment_status.py:47]: Ah, ok. You would be the one looking into the logs after all :)

**Review by mbukatov — DISMISSED** [review]

**Review by jilju — DISMISSED** [review]

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/15165/

**Review by mbukatov — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Pull request merged**